src/pykiso/lib/auxiliaries/zephyr.py

```diff
@@ -0,0 +1,223 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+import enum
+import logging
+import subprocess
+import xml.etree.ElementTree as ET
+from pathlib import Path
+from typing import Optional
+from pykiso import AuxiliaryInterface
+from pykiso.types import MsgType
+
+log = logging.getLogger(__name__)
+
+
+class TestResult(enum.IntEnum):
+    """Result of a twister test"""
+
+    PASSED = 0
+    FAILED = 1
+    SKIPPED = 2
+    ERROR = 3
+
+
+class ZyphyrException(Exception):
+    """Exception for twister errors"""
+
+    def __init__(self, message) -> None:
+        super().__init__(message)
+
+
+class Twister:
```

Comment:
It should still be investigated if this could somehow become a CChannel (just to respect the general architecture, I don't see any other useful reason in doing this)